packages/adapters/src/adapters/Cornerstone/Segmentation_4X.js

```diff
@@ -359,11 +359,11 @@ async function generateToolState(
     // we don't have to call metadataProvider.get() for each imageId over
     // and over again.
     const sopUIDImageIdIndexMap = imageIds.reduce((acc, imageId) => {
-        const { sopInstanceUid } = metadataProvider.get(
+        const { sopInstanceUID } = metadataProvider.get(
```

Comment:
unfortunatelly this also is a bad design and our fault, dcmjs namify generates sopInstanceUid i guess, so we should again handle fallback here too, to see which one has value

Reply:
https://github.com/OHIF/Viewers/pull/4000